File: packages/core/frontend/src/types/resources/priorityclass.ts
import type { ManagedField } from "./shared";

export interface PriorityClass {
  Name: string;
  Value: number;
  GlobalDefault: boolean;
  Description: string;
  PreemptionPolicy: string;
  Age: string;
  CreatedAt: string;
  ManagedFields: ManagedField[];
}

File: packages/core/kube/dto/priorityclass.go
package dto

type PriorityClassRef struct {
	Name string `json:"name"`
}

type PriorityClass struct {
	Name             string
	Value            int32
	GlobalDefault    bool
	Description      string
	PreemptionPolicy string
	Age              string
	CreatedAt        string
	ManagedFields    []ManagedField
}
